src/dscanner/analysis/ifelsesame.d

```diff
@@ -5,111 +5,194 @@
 
 module dscanner.analysis.ifelsesame;
 
-import std.stdio;
-import dparse.ast;
-import dparse.lexer;
 import dscanner.analysis.base;
-import dscanner.analysis.helpers;
-import dsymbol.scope_ : Scope;
+import dmd.hdrgen : toChars;
```

Comment:
ASTNode.toChars() has moved in the dmd.hdrgen module. Are we allowed to import this, or do we need to relex the file? @RazvanN7